<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
	<class>TaskClass</class>
	<widget class="QWidget" name="TaskClass">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>577</width>
				<height>120</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>Task</string>
		</property>
		<layout class="QGridLayout" name="gridLayout">
			<item row="2" column="3">
				<widget class="QLabel" name="qlCategory">
					<property name="text">
						<string>TextLabel</string>
					</property>
				</widget>
			</item>
			<item row="0" column="2">
				<widget class="QPushButton" name="pbDelete">
					<property name="text">
						<string>Delete task</string>
					</property>
				</widget>
			</item>
			<item row="2" column="0" colspan="2">
				<widget class="QLabel" name="qlDaetPeriod">
					<property name="text">
						<string>TextLabel</string>
					</property>
				</widget>
			</item>
			<item row="1" column="2">
				<spacer name="horizontalSpacer">
					<property name="orientation">
						<enum>Qt::Horizontal</enum>
					</property>
					<property name="sizeHint" stdset="0">
						<size>
							<width>427</width>
							<height>20</height>
						</size>
					</property>
				</spacer>
			</item>
			<item row="0" column="3">
				<widget class="QLabel" name="qlImportant">
					<property name="text">
						<string>TextLabel</string>
					</property>
				</widget>
			</item>
			<item row="1" column="1">
				<widget class="QLabel" name="qlTextTask">
					<property name="text">
						<string>TextLabel</string>
					</property>
				</widget>
			</item>
			<item row="0" column="0" colspan="2">
				<widget class="QLabel" name="qlNameTask">
					<property name="text">
						<string>TextLabel</string>
					</property>
					<property name="textFormat">
						<enum>Qt::AutoText</enum>
					</property>
					<property name="scaledContents">
						<bool>false</bool>
					</property>
					<property name="wordWrap">
						<bool>false</bool>
					</property>
				</widget>
			</item>
			<item row="2" column="2">
				<widget class="QPushButton" name="pbEdit">
					<property name="text">
						<string>Edit task</string>
					</property>
				</widget>
			</item>
			<item row="1" column="0">
				<widget class="QCheckBox" name="qcbIsComplete">
					<property name="text">
						<string/>
					</property>
				</widget>
			</item>
		</layout>
	</widget>
	<layoutdefault spacing="6" margin="11"/>
	<resources/>
	<connections/>
</ui>